projects/core/components/expand/expand.component.ts

```diff
@@ -47,6 +45,13 @@ export const TUI_EXPAND_LOADED = 'tui-expand-loaded';
     animations: [tuiParentAnimation],
     host: {
         ngSkipHydration: 'true',
+        '[style.height.px]': 'height',
+        '[class._loading]': 'loading',
+        '[class._overflow]': 'overflow',
+        '[class._expanded]': 'expanded',
+        '[attr.aria-expanded]': 'expanded',
+        '(tui-expand-loaded)': 'onExpandLoaded($event)',
```

Comment:
Could we use const `TUI_EXPAND_LOADED` ?

Reply:
fixed